crates/node/src/service.rs

```diff
@@ -158,6 +184,7 @@ where
 
     let madara_backend = Arc::new(MadaraBackend::open(&config.database, &db_config_dir(config))?);
 
+    // Call the `build_import_queue` function to create an import queue and block import for the service.
```

Comment:
This adds nothing more than what you already know by reading the code.
I think you can remove it